tony-core/src/main/java/com/linkedin/tony/tensorflow/TensorFlowSession.java

```diff
@@ -222,7 +220,7 @@ public synchronized TonyTask getMatchingTask(long allocationRequestId) {
       TonyTask[] tasks = entry.getValue();
       for (int i = 0; i < tasks.length; i++) {
         if (tasks[i] == null) {
-          tasks[i] = new TonyTask(jobName, String.valueOf(i));
+          tasks[i] = new TonyTask(jobName, String.valueOf(i), sessionId);
```

Comment:
This method is currently named `getMatchingTask` but has a side effect of creating a `TonyTask` and updating an array. Can we rename this method `getAndAssignMatchingTask` to indicate that it has a side effect?

Reply:
changed to `getAndInitMatchingTask`